libensemble/tests/scaling_tests/forces/run_libe_forces.py

```diff
@@ -38,12 +38,15 @@
 
 # Create executor and register sim to it.
 if USE_BALSAM:
-    from libensemble.executors.balsam_executor import BalsamMPIExecutor
```

Comment:
Are the changes here (under regular forces directory) for running balsam on the local machine?